README.md

```diff
@@ -8,13 +8,13 @@
 - [Enhancements Tracking Spreadsheet](#enhancements-tracking-spreadsheet)
 - [Labels](#labels)
 
-Enhancement tracking repo for Kubernetes releases. Owned by [SIG Architecture](https://git.k8s.io/community/sig-architecture#enhancements).
+Enhancement tracking repository for Kubernetes releases. Owned by [SIG Architecture](https://git.k8s.io/community/sig-architecture#enhancements).
```

Comment:
```suggestion
Enhancement tracking repository for Kubernetes releases. Owned by [SIG Architecture](https://git.k8s.io/community/sig-architecture#enhancements).
```

Personally, I think repo is fine. It is accepted shorthand used throughout the project and many others.